docs/source/algorithms/LoadGaussCube-v1.rst

```diff
@@ -0,0 +1,31 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Algorithm to load Gaussian Cube files (for volumetric data) into a 3D :ref:`MDHistoWorkspace <MDHistoWorkspace>`.
+The file is assumed to follow the XYZ order for flattening (i.e. the outer loop is over X coordinate and inner loop
+is over Z).
+
+Usage
+-----
+
+**Example - LoadGaussCube**
+
+.. code-block:: python
+
+    from mantid.simpleapi import *
+
+    ws = LoadGaussCube(Filename="gauss_cube_example.cube", Names=['[H,0,0]','[0,K,0]','[0,0,L]'], Frames='HKL,HKL,HKL', Units='rlu,rlu,rlu')
```

Comment:
can we not add a doc test here? If we print something we can say what the expected output is.

Reply:
Can do, but given the output is a single workspace, I'm not sure what that would add over the unit tests?

Reply:
Just an extra layer of testing for us. It also tells users what sort of result to expect, you can use it to highlight key differences to "normal" output. I always feel happier seeing a test in the docs as I can check if it works as the docs expect